docs/deploy/ansible/k3s.md

```diff
@@ -0,0 +1,52 @@
+# K3S
+
+*Playbook Path: [ansible/playbooks/install_k3s.yml](https://github.com/webrecorder/browsertrix-cloud/blob/main/ansible/playbooks/install_k3s.yml)*
+
+This playbook provides an easy way to install Browsertrix Cloud on a Linux box (tested on Rocky Linux 9) 
+It automatically sets up Browsertrix with, Letsencrypt certificates.
```

Comment:
Docs pages should not have manual line wrapping.  I know the file you likely based this off did that, this has since been fixed!  Also there's a bonus comma after "Browsertrix with" that, while fun, should be deleted (also a leftover from the template file, now fixed)